examples/player/simid_player.html

```diff
@@ -78,9 +79,18 @@
         <button onclick="switchCreative('survey.html')">survey</button>
         <button onclick="switchCreative('selector.html')">selector</button>
         <button onclick="switchCreative('extender.html')">extender</button>
+        <button onclick="switchCreative('nonlinear_overlay.html')">nonlinear_overlay</button>
       </td>
       </tr>
-
+      <tr>
+        <td>Ad type</td>
+        <td>
+          <input type="radio" id="linear_ad" name="ad_type" checked> <label for="linear_ad" style="font-size: 16px;">Linear
```

Comment:
Lets make this radio button a little more active. When non linear is selected hide video url.  When in linear mode, show url.